api/src/main/java/org/openmrs/module/ugandaemrreports/reports/SetupActiveOnCareList.java

```diff
@@ -96,14 +97,20 @@ public String getDescription() {
     @Override
 
     public ReportDesign buildReportDesign(ReportDefinition reportDefinition) {
-        ReportDesign rd = createExcelTemplateDesign(getExcelDesignUuid(), reportDefinition, "ActiveoncareList.xls");
+        ReportDesign rd = createExcelTemplateDesign("c2ef160f-f855-462d-a2ad-c38ca3ceffa0", reportDefinition, "ActiveoncareList.xls");
```

Comment:
@solemabrothers Why the change, this seems contrary to other approaches

Reply:
For all reports where we have a CSV export, we need to use a different uuid as using the same Excel UUID throws the duplicate uuid entry error.

Reply:
@solemabrothers why not add a getCSVDesignUuid() method?

Reply:
@solemabrothers how about the getUuid method